report/thesis.tex

```diff
@@ -904,7 +899,7 @@ \chapter{Related problems and future work}
 
 ## Fork detection
 
-A fork detection solution aims to detect a fork as soon as possible in a reliable and efficient way.
+A fork detection solution~\cite{failure-detection} aims to detect a fork as soon as possible in a reliable and efficient way.
 
 We define the following interface for a fork detection algorithm:
 
```

Comment:
Yes I remember we discussed this and I think the section explains pretty much what we have said in simple words, what do you think it is not correct or should be modified in particular?

Reply:
You don't need to have a trusted third party connected to every validator.
Oh, I see that COMMIT = DECISION.
Nevertheless, I would remove the sentence "you need to have a trusted third party..." just to avoid potential questions that may arise.

Reply:
Fixed